modules/ROOT/pages/op-tkn-cli-clustertask-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// *  pipelines/op-tkn-cli-reference.adoc
+
+[id="cli-clustertask-management-commands_{context}"]
```

Comment:
I try to keep the IDs same as the file names, easy to copy paste and also if you need to use ifdef it helps to be consistent. But I prefer the shortened version. As mentioned above we could either keep tkn/cli.

Reply:
done

Reply:
This is not the same as of now, please see my comment.